docs/getting-started/index.md

```diff
@@ -28,34 +28,53 @@ under the License.
 -->
 
 There are many ways to get started with Apache Flink. Which one is the best for
-you depends on your goal and prior experience:
+you depends on your goals and prior experience:
 
-* take a look at the **Docker Playgrounds** for a docker-based introduction to
-  specific Flink concepts
-* explore on of the **Code Walkthroughs** if you want to get an end-to-end
-  introduction to using one of the Flink APIs
-* use **Project Setup** if you already know the basics of Flink but want to get a
-  project setup template for Java or Scala and need help setting up
-  dependencies
+* take a look at the **Docker Playgrounds** if you want to see what Flink can do, via a hands-on,
+  docker-based introduction to specific Flink concepts
+* explore one of the **Code Walkthroughs** if you want a quick, end-to-end
+  introduction to one of Flink's APIs
+* work your way through the **Hands-on Training** for a comprehensive,
+  step-by-step introduction to Flink
+* use **Project Setup** if you already know the basics of Flink and want a
+  project template for Java or Scala, or need help setting up the dependencies
```

Comment:
What do you think about linking these sections directly here? That way, you don't have to look for them in the navigation; especially with the Hands-On Training not being part of this section.

Reply:
I think it might be odd to only link to the Training here, and the other 3 aren't linkable.

Reply:
true - didn't see that the other 3 are not linkable...